Content.Shared/Atmos/Consoles/SharedAtmosMonitoringConsoleSystem.cs

```diff
@@ -15,7 +15,7 @@ public override void Initialize()
 
     private void OnGetState(EntityUid uid, AtmosMonitoringConsoleComponent component, ref ComponentGetState args)
     {
-        Dictionary<Vector2i, Dictionary<(int, string), ulong>> chunks;
+        Dictionary<Vector2i, Dictionary<(int, byte, string), ulong>> chunks;
```

Comment:
This tuple `(int, byte, string)` is used in so many different places. It deserves a named type. See DRY principle